app/renderer/views/Page/Footer.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import styled from 'react-emotion'
+import theme from '@/style/common'
+
+const Container = styled.footer`
```

Comment:
I don't really like creating a separate file when there is no real necessity - it's nice having the smaller components and style next to the JSX, part of the idea of css on js I think. Not critical though.